_data/team.yml

```diff
@@ -104,3 +104,19 @@
   network:
     - name: linkedin
       link: https://www.linkedin.com/in/madhurachaganty
+
+- name: Silke Nodwell
+  position: WCC Lead
+  comment: Data Scientist
+  image: assets/images/team/12.jpeg
+  network:
+    - name: linkedin
+      link: https://www.linkedin.com/in/silke-nodwell-763681172
+
+- name: Abhipsa Mishra
+  position: WCC Lead
```

Comment:
Please could you move to the order the member joined the team?
* Abhipsa to after Elena 
* Silke to after Arzu 

I think also is missing `Ekaterine Oshkhereli - Leader` If possible to do as part of this one. Or can be another one.

Reply:
Also added Sonali with this PR:
<img width="767" alt="image" src="https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/134794618/7b73a676-15b2-4f7f-8d4d-a8a5eb5d35c7">